d8s_python/python_data.py

```diff
@@ -250,9 +250,12 @@ def python_copy_shallow(python_object: Any) -> Any:
 # @decorators.map_first_arg
 def python_file_names(path: str, *, exclude_tests: bool = False) -> List[str]:  # noqa: CCR001
```

Comment:
The [type hint](https://docs.python.org/3/library/typing.html) for the return type of this function should be updated. Previously, this function returned a list. Now, as you can see [here](https://github.com/democritus-project/d8s-python/pull/31/files#diff-69fdfc13e468155410f46b17d8c31c54c12261bb65e349eae87b753928044c5fR261), it returns a generator (because this is what the [pathlib.Path.glob function](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob) returns). Does that make sense?

```suggestion
def python_file_names(path: str, *, exclude_tests: bool = False) -> Iterator[str]:  # noqa: CCR001
```